Formula/rustup-init.rb

```diff
@@ -23,17 +23,11 @@ class RustupInit < Formula
   end
 
   def install
-    cargo_home = buildpath/"cargo_home"
-    cargo_home.mkpath
-    ENV["CARGO_HOME"] = cargo_home
-
     system "cargo", "install", "--features", "no-self-update", *std_cargo_args
   end
 
   test do
-    ENV["CARGO_HOME"] = testpath/".cargo"
```

Comment:
I think we can keep this. I think the test sets it so that it know where to call `rustc` later on.